content/testing/fluent-assertions.md

```diff
@@ -0,0 +1,96 @@
+---
+title: FluentAssertions
+showMetadata: true
+editable: true
+showToc: true
+---
+
+สมมติว่าเรามี Data Structure ดังนี้ คำถามคือเราจะ Assert อย่างไร
+
+```c#
+public class Todo
+{
+    public int id  { get; set; }
+    public string title  { get; set; }
+    public bool completed  { get; set; }
+}
+
+public class TodoList
+{
+
+    public IEnumerable<Todo> GetTodoList()
+    {
+        return new Todo[]
+        {
+            new Todo{
+                id = 0,
```

Comment:
Very minor if it is the id of an object, please use a real word/business/db that start from 1. 
I think this id is not an idexer.

Reply:
Minor for me, it’s not necessary to change. Not a standard. It depends on platforms

Reply:
For this one, I insist to follow a language naming convention which is not a platform specific.
If we use Java, JS or TS, I agree that we use camel case for a property/getter/setter.

Here is the reference, it is better to follow a convention. When we work with other .NET people or contribute to open source project. We use the same pattern.

https://docs.microsoft.com/en-us/dotnet/csharp/fundamentals/coding-style/coding-conventions
```
public class ExampleEvents
{
    // A public field, these should be used sparingly
    public bool IsValid;

    // An init-only property
    public IWorkerQueue WorkerQueue { get; init; }

    // An event
    public event Action EventProcessing;

    // Method
    public void StartEventProcessing()
    {
        // Local function
        static int CountQueueItems() => WorkerQueue.Count;
        // ...
    }
}
```

Reply:
I think you may confuse, i understand .net convention, however  we're discussing about indexing start with 0 or 1.

Reply:
My meaning is index is nothing to define the object data in this case

Reply:
@mildronize  Oh mann how I lost the context, I thought here is the comment for naming convention.
My big mistake, so we agree to follow naming convention and you have updated it.
So sorry again.

Yeah this one we can leave as it is but this remind me for something like we should start page index from 0 or 1 LoL.